jobs/build/ocp/Jenkinsfile

```diff
@@ -133,6 +139,43 @@ ${OA_CHANGELOG}
     }
 }
 
+// extract timing information from the record_log and write a report string
+// the timing record log entry has this form:
+// image_build_metrics|elapsed_total_minutes={d}|task_count={d}|elapsed_wait_minutes={d}|
+def get_build_timing_report() {
+
+    record_log = buildlib.parse_record_log(OIT_WORKING)
+    metrics = record_log['image_build_metrics']
+    if metrics.size == 0 {
+	return ""
+    }
+
+    // metrics[0]['task_count']
+    // metrics[0]['elapsed_total_minutes']
+    // metrics[0]['elapsed_wait_minutes']
+
+    return """
+Images built: ${metrics[0]['task_count']}
+Elapsed Time: ${metrics[0]['elapsed_total_minutes']} minutes
+OSBS Resource Wait time: ${metrics[0]['elapsed_wait_minutes']} minutes
+""".trim()
+}
+
+// get the list of images built
+def get_image_build_report() {
+
+    record_log = buildlib.parse_record_log(OIT_WORKING)
+    builds = record_log['build']
+    image_list = []
+
+    for (i = 0; i < builds.size(); i++) {
```

Comment:
Since the same record log can be used for multiple resumes, you will need to remove duplicates and check for status=0, push_status=0
`build|status=0|push_status=0|....`
Alphabetizing end result would also help folks.